hack/hypernetes/util.sh

```diff
@@ -36,8 +36,12 @@ function kube::util::verify_system() {
 
 function kube::util::setup_ssh() {
 	echo "Start $FUNCNAME"
-	ssh-keygen -f /root/.ssh/id_rsa -t rsa -N ''
-	cat /root/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys
+	local key_path="/root/.ssh/id_rsa"
+        if [[ -e ${key_path} ]] ; then
+		rm -f ${key_path}
```

Comment:
This is a very dangerous operation, we shouldn't remove the ssh key. Instead, if the key already exist, just jump to `authorized_keys` and `ssh-keyscan` step.